packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -51,248 +47,6 @@ QUnit.module('ember-htmlbars: {{#if}} and {{#unless}} helpers', {
   }
 });
 
-QUnit.test('The `if` helper tests for `isTruthy` if available', function() {
-  view = EmberView.create({
-    truthy: EmberObject.create({ isTruthy: true }),
-    falsy: EmberObject.create({ isTruthy: false }),
-
-    template: compile('{{#if view.truthy}}Yep{{/if}}{{#if view.falsy}}Nope{{/if}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'Yep');
-});
-
-QUnit.test('The `if` helper does not error on undefined', function() {
-  view = EmberView.create({
-    undefinedValue: undefined,
-    template: compile('{{#if view.undefinedValue}}Yep{{/if}}{{#if (unbound view.undefinedValue)}}Yep{{/if}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), '');
-});
-
-QUnit.test('The `unless` helper does not error on undefined', function() {
-  view = EmberView.create({
-    undefinedValue: undefined,
-    template: compile('{{#unless view.undefinedValue}}YepBound{{/unless}}{{#unless (unbound view.undefinedValue)}}YepUnbound{{/unless}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'YepBoundYepUnbound');
-});
-
-QUnit.test('The `if` helper does not print the contents for an object proxy without content', function() {
```

Comment:
Replaced by:

* "it considers object proxies without content falsy"

(Now testing re-render, mutation and replacement as well)